suites/reef/rados/tier-2_rados_test-pg-dups-trimming.yaml

```diff
@@ -0,0 +1,203 @@
+# Suite is to be used to verify the dups trimming scenario on all NVMe cluster
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574887
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                # using 6.1 rc candidate
+                custom_image: "registry-proxy.engineering.redhat.com/rh-osbs/rhceph:6-177"
```

Comment:
We would not be able to reproduce the customer issue with the mentioned build. We would need to update the build here before this test is included in the baremetal pipeline.